scania-digital-design-system/tegel
# feat(icons): added history, flash, trailor

## **Describe pull-request**  


Adding 3 icons into tegel pacakge, that exist in industrial icons (in the figma-ui kit)


## **How to test**  
_Provide detailed steps for testing, including any necessary setup._
1. Go to Icons page in storybook
2. Check the icons that added in this PR
3. Use them

## **Checklist before submission**

## **Suggested test steps**

## **Screenshots**  
_Include before/after screenshots for UI changes._

## **Additional context**  
_Add any other context or feedback requests about the pull-request here._
